src/NuGet.Core/NuGet.Protocol/Model/PackageSearchMetadataV2Feed.cs

```diff
@@ -85,6 +84,8 @@ public PackageSearchMetadataV2Feed(V2FeedPackageInfo package, MetadataReferenceC
 
         public Uri LicenseUrl { get; private set; }
 
+        public List<string> OwnersList => null;
```

Comment:
A considerable number of customers still use nuget.org's V2 URL.  Will future PM UI changes not work for these customers that use V2 feeds because of this?

Reply:
V2 does not currently return Owners like V3 does. 

I checked with the server team and it seems they do not backport features to V2. So, I'm not sure I see a problem for future PM UI, since the existing behavior will not change. That is, the package source nuget.org V2 will never return Owners.

Reply:
Owners was added to both the nuspec, and also NuGet.Client's internal data model before NuGet went 1.0: https://github.com/NuGet/NuGet2/commit/a6f17b03736ba178a21fe422603442eff8ee8d65

Did you validate that the V2 protocol doesn't return owners?  It would really surprise me if it's not already being returned. After all, the concept of owners has existed ever since nuget.org has.

Reply:
Yes, I know you're not available offline, but if you get a moment, I can send you a sample response from nuget.org that I captured with Fiddler. There's no Owners. 

`OwnersList` is not intended to represent `Owners` from the nuspec. It's only a Search API result of `owners` from https://learn.microsoft.com/en-us/nuget/api/search-query-service-resource#search-result.
The code you linked is for a nuspec Owner, which is not the same thing.